### ISSUE TITLE
valid-exports: false positive for `null` subpath patterns

### ISSUE THREAD

**Issue by Cellule** [JoshuaKGoldberg/eslint-plugin-package-json#1790]
## Bug report

The `package-json/valid-exports` rule reports an error for export entries whose value is `null`, but `null` is a valid value in Node.js `exports` and is the documented way to explicitly disallow access to subpaths.

### Repro

```json
{
  "name": "example",
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    },
    "./*": null
  }
}
```

### Actual

```
error  Invalid exports: the value of "./*" should be either an entry point path or an object of export conditions  package-json/valid-exports
```

### Expected

No error. According to Node.js docs, a subpath value of `null` is valid and explicitly blocks the matching subpath:

> A package can also block access to its modules by exporting a key with a value of `null`.

References:
- https://nodejs.org/api/packages.html#subpath-patterns (search for `null`)
- https://nodejs.org/api/packages.html#exports

### Notes

- The current docs for `valid-exports` only mention `string` or `object` as valid values; `null` should be added as a valid value to disallow a subpath.
- The recommended config currently makes it impossible to use this Node.js feature without disabling the rule.

### Versions

- `eslint-plugin-package-json`: latest
- Node.js: 22+

**Comment by Cellule** [comment]
Wait I think I was a bit too hasty, looks like this might not be a problem on latest version I'll test again

**Comment by michaelfaith** [comment]
> Wait I think I was a bit too hasty, looks like this might not be a problem on latest version I'll test again

This should have been addressed in https://github.com/michaelfaith/package-json-validator/issues/732.

**Comment by Cellule** [comment]
yes my bad! 🙏 I've confirmed it's fixed on latest.
Sorry!
